container/load.bzl

```diff
@@ -0,0 +1,58 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Rule for importing an image from 'docker save' tarballs.
+
+This extracts the tarball, examines the layers and creates a
+container_import target for use with container_image.
+"""
+
+def _container_load_impl(ctx):
+  result = ctx.execute(["mkdir", "image"])
+  if result.return_code:
+    fail("Creating directory failed (status %s): %s" % (result.return_code, result.stderr))
+
+  result = ctx.execute([
+      ctx.path(ctx.attr._importer),
```

Comment:
For portability to Windows we need to prefix `PAR` files with the `python` path.  That's what [this](https://github.com/bazelbuild/rules_docker/blob/master/container/pull.bzl#L21) is doing in `container_pull`.

Sadly we haven't reached the point where this is on our Bazel CI config, but I'd like to avoid regressing here.  You can rename this (drop leading underscore) and then import it from `pull.bzl`.  I can worry about finding it a better home later.

Reply:
Done.